yew-macro/tests/html_macro_test.rs

```diff
@@ -0,0 +1,46 @@
+use yew::html;
+
+#[allow(dead_code)]
+#[rustversion::attr(stable(1.45), test)]
+fn html_macro() {
+    let t = trybuild::TestCases::new();
+
+    t.pass("tests/html_macro/html-block-pass.rs");
+    t.compile_fail("tests/html_macro/html-block-fail.rs");
+
+    t.pass("tests/html_macro/html-component-pass.rs");
+    t.compile_fail("tests/html_macro/html-component-fail.rs");
+    t.compile_fail("tests/html_macro/html-component-fail-unimplemented.rs");
+
+    t.pass("tests/html_macro/html-iterable-pass.rs");
+    t.compile_fail("tests/html_macro/html-iterable-fail.rs");
+
+    t.pass("tests/html_macro/html-list-pass.rs");
+    t.compile_fail("tests/html_macro/html-list-fail.rs");
+
+    t.pass("tests/html_macro/html-node-pass.rs");
+    t.compile_fail("tests/html_macro/html-node-fail.rs");
+
+    t.pass("tests/html_macro/html-element-pass.rs");
+    t.compile_fail("tests/html_macro/html-element-fail.rs");
```

Comment:
while we're at it, let's switch to the wildcard approach for these:
```suggestion
    t.pass("tests/html_macro/*-pass.rs");
    t.compile_fail("tests/html_macro/*-fail.rs");
```

The only problem is the `html-component-fail-unimplemented.rs` fail, let's just rename that one to `html-component-unimplemented-fail.rs` so it matches the pattern.